### DISCUSSION TITLE
RuCore / Системы именования / Свидетельство о рождении

### DISCUSSION THREAD

**Discussion by eugenekogan-spb** [fhir-ru/core#247] [Минимальный набор артефактов]
//--------------------------------------
//страница: Руководство RuCore / Системы именования / Государственное свидетельство о рождении
//--------------------------------------

```
Instance:   core-ns-rf-birth-certificate
InstanceOf: NamingSystem
Usage: #definition

* name =   "Core_Ns_Rf_Birth_Certificate"

* description = "Номера государственных свидетельств о рождении формируются государственной системой органов записи актов гражданского состояния (ЗАГС), с внесением в единый реестр записей актов гражданского состояния (ЕГР ЗАГС). Предполагается, что номер уникален, сведений об ограничении срока уникальности номера нет."

* status = #active

* kind = #identifier

* date = "2023-11-22"

* uniqueId
  * type = #uri
  * value = "http://fhir.ru/core/systems/birth-certificate"
  * preferred = true
```

**Comment by Vrodvi** [comment]
# Замечание 1.

Вот тут https://github.com/fhir-ru/core/discussions/235#discussioncomment-7639539 описаны пояснения по ведению данного пространства и гарантиям уникальности идентификаторов в нём.
Необходимо внести эти пояснения в текстовое описание ресурса, для этого подойдёт атрибут description:
![image](https://github.com/fhir-ru/core/assets/92928137/b41554a8-e786-4810-a220-4a665574d9cd)

**Reply by eugenekogan-spb** [reply]
Пояснения добавлены. На страничку метрекомендации добавлена рекомендация.

**Reply by Vrodvi** [reply]
Замечание "1." закрыто.
